package.json

```diff
@@ -42,29 +43,28 @@
     "co-wait": "0.0.0",
     "docker-raw-stream": "^1.3.1",
     "dockerode": "^2.0.4",
-    "eslint-config-probo": "^1.0.2",
+    "eslint-config-probo": "^2.0.0",
     "event-stream": "^3.3.2",
     "express": "^4.12.3",
     "flake-idgen": "^1.0.0",
-    "github": "^0.2.3",
-    "github-api": "^0.10.3",
-    "github-client": "^0.8.0",
-    "github-webhook-handler": "^0.3.4",
-    "glob": "^4.4.2",
+    "github-webhook-handler": "^0.7.1",
+    "glob": "^7.1.3",
     "http-proxy": "^1.8.1",
     "js-yaml": "^3.2.7",
-    "level": "^1.3.0",
-    "lodash": "^3.10.1",
+    "kafka-node": "^3.0.0",
+    "level": "^5.0.1",
+    "lodash": "^4.17.11",
     "ms": "^0.7.1",
-    "probo-eventbus": "^0.1.1",
+    "probo-eventbus": "^1.0.4",
     "probo-request-logger": "^1.0.1",
     "redis": "^0.12.1",
     "request": "^2.53.0",
-    "request-promise": "^1.0.2",
-    "restify": "^2.8.5",
+    "request-promise": "^4.2.2",
+    "restify": "^8.3.2",
+    "restify-errors": "^8.0.0",
```

Comment:
Restify move the error classes to a different package. That's why this new package `restify-errors` is included.